### DISCUSSION TITLE
VO-specific information in the main Rucio repository

### DISCUSSION THREAD

**Discussion by rdimaio** [rucio/rucio#6576] [General]
I might be misunderstanding what a VO represents, but I'm a bit confused by the presence of VO-specific schemas here: https://github.com/rucio/rucio/tree/master/lib/rucio/common/schema

I understand the presence of VO-specific information in the tests, for testing purposes, but should the source code have VO-specific information too?

**Comment by rdimaio** [comment]
From @bari12 on Mattermost (duplicating here for future reference):
> This is a bit of an idiosyncrasy from the past and we should probably re-do how we handle these things.
Only a few of the "configurations" in this file actually make sense (let's say the regexp for rse names, etc.) But most of them I doubt if they should even exist like this
> We will push these files out of the upstream code though, they should only be there via policy packages, but even then they exist in this form, which probably should be scaled down

**Comment by voetberg** [comment]
I wonder if this can be a larger effort to start removing any vo-specifics from the main repo? It would be good to plan for stripping out all the legacy vo stuff at some point.
